pom.xml

```diff
@@ -1500,10 +1500,8 @@
             <!-- The next two lines force use of the local lib/openlcb.jar file. -->
             <!-- This is needed when we're using a custom openlcb.jar file -->
             <!-- not yet available on a maven repository. -->
-<!-- 
```

Comment:
This change causes problems with NetBeans. See [comment](https://github.com/JMRI/JMRI/pull/12606#issuecomment-1808460683). NetBeans uses maven to compile JMRI.

I don't remember what the problem was, but I think it's bad to remove this comment.

Reply:
The problem is that OpenLCB needed to change the signature of a method, which has to be synchronized with the corresponding change to JMRI's use of it in order to use the new library.  For Ant builds, no problem, those use the local openlcb.jar file from lib/.  

Maven builds, like CI, normally do _not_ use the local lib/ jar files. They take from the repository.  If OpenLCB has not successfully put out a new release with the change, then Maven builds don't build  unless those two lines are un-commented out to tell Maven to use the local file.

What goes wrong in NetBeans?

Reply:
> What goes wrong in NetBeans?

Maven gives this warning which confuses developers:
```
Some problems were encountered while building the effective model for org.jmri:jmri:jar:5.9.3-SNAPSHOT
'dependencies.dependency.systemPath' for org.openlcb:openlcb:jar should not point at files
within the project directory, ${project.basedir}/lib/openlcb.jar will be unresolvable by
dependent projects @ line 1504, column 25

It is highly recommended to fix these problems because they threaten the stability of your build.

For this reason, future Maven versions might no longer support building such malformed projects.
```

.

> The problem is that OpenLCB needed to change the signature of a method, which has to be synchronized with the corresponding change to JMRI's use of it in order to use the new library.

I understand that. But how long time will it take for the new OpneLCB version to get into Maven repository? If we are talking about a week or two, I think it would be better to wait to merge this PR until the new version is in the Maven repository.

Reply:
I have given this a second thought and I think it's OK to merge this PR now if this change is reverted once the new version of the lib is in the Maven repository.